CONTRIBUTING.md

```diff
@@ -0,0 +1,217 @@
+# Contributing
+
+We welcome contributions from the community. Please read the following guidelines carefully to
+maximize the chances of your PR being merged.
+
+## Communication
+
+* Before starting work on a major feature, please reach out to us via GitHub, Slack,
+  email, etc. We will make sure no one else is already working on it and ask you to open a
+  GitHub issue.
+* A "major feature" is defined as any change that is > 100 LOC altered (not including tests), or
+  changes any user-facing behavior. We will use the GitHub issue to discuss the feature and come to
+  agreement. This is to prevent your time being wasted, as well as ours. The GitHub review process
+  for major features is also important so that [organizations with commit access](OWNERS.md) can
+  come to agreement on design. If it is appropriate to write a design document, the document must
+  be hosted either in the GitHub tracking issue, or linked to from the issue and hosted in a
+  world-readable location.
+* Small patches and bug fixes don't need prior communication.
+
+## Inclusive language policy
+
+The Envoy community has an explicit goal to be inclusive to all. As such, all PRs must adhere to the
+following guidelines for all code, APIs, and documentation:
+
+* The following words and phrases are not allowed:
+  * *Whitelist*: use allowlist instead.
+  * *Blacklist*: use denylist or blocklist instead.
+  * *Master*: use primary instead.
+  * *Slave*: use secondary or replica instead.
+* Documentation should be written in an inclusive style. The [Google developer
+  documentation](https://developers.google.com/style/inclusive-documentation) contains an excellent
+  reference on this topic.
+* The above policy is not considered definitive and may be amended in the future as industry best
+  practices evolve. Additional comments on this topic may be provided by maintainers during code
+  review.
+
+## Breaking change policy
+
+Both API and implementation stability are important to Envoy. Since the API is consumed by clients
```

Comment:
I think this section does not apply to go-control-plane as it is speaking about xDS versioning policy. Since we don't have versioned APIs defined strictly here, I think we can omit this for now.

Reply:
Sounds good will omit